aws/resource_aws_cloudtrail.go

```diff
@@ -116,7 +118,21 @@ func resourceAwsCloudTrailCreate(d *schema.ResourceData, meta interface{}) error
 		input.SnsTopicName = aws.String(v.(string))
 	}
 
-	t, err := conn.CreateTrail(&input)
+	var t *cloudtrail.CreateTrailOutput
+	err := resource.Retry(15*time.Second, func() *resource.RetryError {
+		var err error
+		t, err = conn.CreateTrail(&input)
+		if err != nil {
+			if isAWSErr(err, "InvalidCloudWatchLogsRoleArnException", "Access denied.") {
```

Comment:
It's 17 extra characters! :trollface: but trolling aside - yes, constant makes sense in this context and it may help avoiding typos. I'll change it.